web/pages/charity/index.tsx

```diff
@@ -3,26 +3,41 @@ import { useState, useMemo } from 'react'
 import { charities as charityList } from '../../../common/charity'
 import { CharityCard } from '../../components/charity/charity-card'
 import { Col } from '../../components/layout/col'
+import { Spacer } from '../../components/layout/spacer'
 import { Page } from '../../components/page'
 import { Title } from '../../components/title'
-
-const charities = charityList.map((charity) => ({
-  ...charity,
-  raised: 4001,
-}))
+import { useAllCharityTxns } from '../../hooks/use-charity-txns'
 
 export default function Charity() {
+  const allCharityTxn = useAllCharityTxns()
```

Comment:
I suppose we could even pass the raised amount into the cards, instead of re-fetching the txns in each card.

But really it doesn't matter. This works fine!